README.md

```diff
@@ -1 +1,38 @@
-# Capston_Project
\ No newline at end of file
+# Capston_Project
+
+- Mobile ScreenShot
+-![screenshot](./assets/images/mobile.png)
+- Desktop ScreenShot
+-![screenshot](./assets/images/disktop.png)
+- [Video Demo](https://zuhra-hashimi.github.io/Capston_Project/).
```

Comment:
This is the link to the live demo. Please try recording a video using Loom and update the link.